src/main/java/org/kairosdb/datastore/cassandra/cache/DefaultMetricNameCache.java

```diff
@@ -0,0 +1,15 @@
+package org.kairosdb.datastore.cassandra.cache;
+
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+import org.kairosdb.datastore.cassandra.cache.persistence.GeneralHashCacheStore;
+
+public class DefaultMetricNameCache extends AbstractStringCache implements StringKeyCache {
```

Comment:
`AbstractStringCache` already implements `StringKeyCache`. Do you need to repeat it here?